ajax/libs/countly-sdk-web/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "countly-sdk-web",
+  "version": "15.08.0",
+  "description": "Countly Web SDK",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Countly/countly-sdk-web"
+  },
+  "keywords": [
+    "Countly",
+    "Analytics",
+    "Web"
+  ],
+  "author": "Countly <hello@count.ly> (https://count.ly/)",
+  "license": "https://github.com/Countly/countly-sdk-web/blob/master/LICENSE",
```

Comment:
Would you please what kind of license is this? A customized license? Or a well known public license? Thanks.